assets/dev/js/editor/components/browser-import/files/readers/json.js

```diff
@@ -8,6 +8,10 @@ export class Json extends FileReaderBase {
 		return 'json';
 	}
 
+	static isActive() {
+		return elementor.config.user.is_administrator || ( elementor.config.user.restrictions?.includes( 'json-upload' ) ?? false );
+	}
+
```

Comment:
If the default is true so maybe?

```suggestion
	static isActive() {
		return elementor.config.user.is_administrator || ( elementor.config.user.restrictions?.includes( 'json-upload' ) ?? true );
	}

```

Reply:
The default is false, that's why it's like it is.

Reply:
In file-reader-base.js the default is true, is it not the abstraction?